include/kremlin/internal/types.h

```diff
@@ -58,7 +58,8 @@ typedef const char *Prims_string;
 typedef __m128i FStar_UInt128_uint128;
 #elif !defined(KRML_VERIFIED_UINT128) && !defined(_MSC_VER) && \
       (defined(__x86_64__) || defined(__x86_64) || defined(__aarch64__) || \
-      (defined(__powerpc64__) && defined(__LITTLE_ENDIAN__)))
```

Comment:
With this getting more and more complex we should maybe add a custom define for this so we have only one place where this is specified.